tests/test_error_handling.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import json
+from test_plus import APITestCase
+
+from wazimap_ng.utils import error_handler
+from rest_framework import status
+
+
+@pytest.mark.django_db
+class TestErrorResponseData(APITestCase):
+
+    def test_error_json(self):
+        request = self.get('/foo/bar')
+
+        response = error_handler(request)
```

Comment:
this is a unit test since it's calling the `error_handler` method and should go in `test_utils.py`